# HG changeset patch
# User John W. Eaton <jwe@octave.org>
# Date 1390551886 18000
#      Fri Jan 24 03:24:46 2014 -0500
# Branch stable
# Node ID d9f706dd78df87bd18ad1fed20b268f896cfa2db
# Parent  adb7c7e6a4a19ec418f58080c9b611cdd4d8b5cf
make mkoctfile.m list of variables consistent with mkoctfile.cc (bug #41296)

* mkoctfile.m: Update doc string.

diff --git a/scripts/miscellaneous/mkoctfile.m b/scripts/miscellaneous/mkoctfile.m
--- a/scripts/miscellaneous/mkoctfile.m
+++ b/scripts/miscellaneous/mkoctfile.m
@@ -73,40 +73,43 @@
 ## (or .mex if @samp{--mex} is specified) unless linking
 ## a stand-alone executable.
 ##
 ## @item  -p VAR
 ## @itemx --print VAR
 ## Print the configuration variable VAR@.  Recognized variables are:
 ##
 ## @example
-##    ALL_CFLAGS                FFTW3F_LIBS
-##    ALL_CXXFLAGS              FLIBS
-##    ALL_FFLAGS                FPICFLAG
-##    ALL_LDFLAGS               INCFLAGS
-##    BLAS_LIBS                 LAPACK_LIBS
-##    CC                        LDFLAGS
-##    CFLAGS                    LD_CXX
-##    CPICFLAG                  LD_STATIC_FLAG
-##    CPPFLAGS                  LFLAGS
-##    CXX                       LIBOCTAVE
-##    CXXFLAGS                  LIBOCTINTERP
-##    CXXPICFLAG                LIBS
-##    DEPEND_EXTRA_SED_PATTERN  OCTAVE_LIBS
-##    DEPEND_FLAGS              OCTAVE_LINK_DEPS
-##    DL_LD                     OCT_LINK_DEPS
-##    DL_LDFLAGS                RDYNAMIC_FLAG
-##    EXEEXT                    READLINE_LIBS
-##    F77                       SED
-##    F77_INTEGER_8_FLAG        XTRA_CFLAGS
-##    FFLAGS                    XTRA_CXXFLAGS
-##    FFTW3_LDFLAGS
-##    FFTW3_LIBS
-##    FFTW3F_LDFLAGS
-##
+##    ALL_CFLAGS                  INCFLAGS
+##    ALL_CXXFLAGS                INCLUDEDIR
+##    ALL_FFLAGS                  LAPACK_LIBS
+##    ALL_LDFLAGS                 LD_CXX
+##    AR                          LDFLAGS
+##    BLAS_LIBS                   LD_STATIC_FLAG
+##    CC                          LFLAGS
+##    CFLAGS                      LIBDIR
+##    CPICFLAG                    LIBOCTAVE
+##    CPPFLAGS                    LIBOCTINTERP
+##    CXX                         LIBS
+##    CXXFLAGS                    OCTAVE_HOME
+##    CXXPICFLAG                  OCTAVE_LIBS
+##    DEPEND_EXTRA_SED_PATTERN    OCTAVE_LINK_DEPS
+##    DEPEND_FLAGS                OCTAVE_LINK_OPTS
+##    DL_LD                       OCTAVE_PREFIX
+##    DL_LDFLAGS                  OCTINCLUDEDIR
+##    F77                         OCTLIBDIR
+##    F77_INTEGER8_FLAG           OCT_LINK_DEPS
+##    FFLAGS                      OCT_LINK_OPTS
+##    FFTW3F_LDFLAGS              RANLIB
+##    FFTW3F_LIBS                 RDYNAMIC_FLAG
+##    FFTW3_LDFLAGS               READLINE_LIBS
+##    FFTW3_LIBS                  SED
+##    FFTW_LIBS                   SPECIAL_MATH_LIB
+##    FLIBS                       XTRA_CFLAGS
+##    FPICFLAG                    XTRA_CXXFLAGS
 ## @end example
 ##
 ## @item --link-stand-alone
 ## Link a stand-alone executable file.
 ##
 ## @item --mex
 ## Assume we are creating a MEX file.  Set the default output extension
 ## to ".mex".
